pkg/feature-gates/feature-gates_test.go

```diff
@@ -65,7 +60,19 @@ func createFeatureGates(objects ...runtime.Object) *FeatureGates {
 	objs = append(objs, objects...)
 
 	// Append empty CDIConfig object that normally is created by the reconcile loop
-	cdiConfig := controller.MakeEmptyCDIConfigSpec(common.ConfigName)
+	cdiConfig := &cdiv1.CDIConfig{
```

Comment:
Am I missing something here, but is this not identical to what MakeEmptyCDICOnfigSpec returns?

Reply:
yes, but because MakeEmptyCDICOnfigSpec util is in controller, it will introduce cyclic import of packages feature-gates and controller, at least in tests,